docs/user-guide/install.md

```diff
@@ -1,10 +1,11 @@
 # Installing RVIC
 
 ## Dependencies
-- [Python 2.7](http://www.python.org/)
+- [Python 2.7 or later including Python3](http://www.python.org/)
 - [Numpy](http://www.numpy.org)
 - [Scipy](http://www.scipy.org/)
 - [netcdf4-python](https://code.google.com/p/netcdf4-python/)
+- [Pandas](http://pandas.pydata.org/)
```

Comment:
This and previous lines - if you want to be consistent to with how the groups capitalize their packages, then I think it should be:
 * NumPy
 * SciPy
 * pandas

Also, I think you can remove the trailing `/` from all the links